src/core/server/bootstrap.ts

```diff
@@ -61,15 +61,15 @@ export async function bootstrap({ configs, cliArgs, applyConfigOverrides }: Boot
 
   function reloadLoggingConfig() {
```

Comment:
maybe the name of the function should be changed too?

Reply:
That would make sense, yea 😅